src/cdk/clipboard/copy-to-clipboard.ts

```diff
@@ -21,18 +21,23 @@ import {Clipboard} from './clipboard';
 @Directive({
   selector: '[cdkCopyToClipboard]',
   host: {
-    '(click)': 'doCopy()',
+    '(click)': 'copy()',
   }
 })
 export class CdkCopyToClipboard {
   /** Content to be copied. */
   @Input('cdkCopyToClipboard') text = '';
 
+  /**
+   * Emits when some text is copied to the clipboard. The
+   * emitted value indicates whether copying was successful.
+   */
   @Output() copied = new EventEmitter<boolean>();
 
-  constructor(private readonly clipboard: Clipboard) {}
+  constructor(private readonly _clipboard: Clipboard) {}
 
-  doCopy() {
-    this.copied.emit(this.clipboard.copy(this.text));
+  /** Copies the current text to the clipboard. */
+  copy() {
```

Comment:
I renamed this to align it with the service and the `PendingCopy`, but I'm not totally sure whether it should be public in the first place.

Reply:
I think it's fine being public